addons/escoria-core/game/core-scripts/log/esc_logger.gd

```diff
@@ -54,33 +56,64 @@ func _init():
 		File.WRITE
 	)
 
+	# Use look-ahead/behind to capture the term in braces
+	globals_regex = RegEx.new()
+	globals_regex.compile("(?<=\\{)(.*)(?=\\})")
+
+# Look to see if any globals (names in braces) should be interpreted
+#
+# #### Parameters
+#
+# * string: Text to log
+func _replace_globals(string: String):
```

Comment:
We know this function returns String at all times. It would be nice to explicitely indicate this for later optimization improvements in GDScript.
```suggestion
func _replace_globals(string: String) -> String:
```